### DISCUSSION TITLE
ETA on OBS 28?

### DISCUSSION THREAD

**Discussion by NOT-HAL9000** [obsproject/obs-studio#7024] [General]
I'm sorry if this was the wrong place to ask this, but I'm just wondering if the team has any date in mind for the OBS 28 release? 

Thanks!

**Comment by ElectronicWar** [comment]
v28 is in Beta right now. Beta+RC phase is typically 1-2 months depending on feedback and found issues.

**Reply by NOT-HAL9000** [reply]
Thanks for the prompt reply! I'm looking forward to it!
